examples/create_cutout.py

```diff
@@ -1,24 +1,24 @@
-
-# In this example we assume you have set in config.py
-
-# ncep_dir = '/path/to/weather_data/'
-
-# where the files have format e.g.
-
-# 'ncep_dir/{year}{month:0>2}/tmp2m.*.grb2'
-
+# In this example we assume you have
+# * Installed the Copernicus Climate Data Store 'cdsapi' package
+# * Registered and setup your cdsapi key as described here https://cds.climate.copernicus.eu/api-how-to
+# 
+# while atlite does support other data sources, using ERA5 from the CDS
+# is good for a quick and uncomplicated start.
+
+# Register a logger
 import logging
 logging.basicConfig(level=logging.DEBUG)
 
 import atlite
 
+# Define the cutout; this will not yet trigger any major operations
 cutout = atlite.Cutout(name="europe-2011-01",
-                       module="ncep",
-                       xs=slice(-12.18798349, 41.56244222),
-                       ys=slice(71.65648314, 33.56459975),
-                       years=slice(2011, 2011),
-                       months=slice(1,1))
-
-#this is where all the work happens - it took 105 minutes on FIAS'
-#beast resi, with 16 cores; the resulting cutout takes 57 GB
+                       cutout_dir="./",
+                       module="era5",
+                       x=slice(-12.18798349, 41.56244222),
+                       y=slice(71.65648314, 33.56459975),
+                       time=slice("2011-01","2011-01")
```

Comment:
```suggestion
                       time="2011-01"
```

Reply:
In the interest of a simple examples: Ok.
I just felt it important to showcase this functionality in the example.